run_dir/static/js/data_del_plot.js

```diff
@@ -0,0 +1,357 @@
+window.current_plot_data = null;
+window.current_plot_obj = null;
+window.current_view_schemes = null;
+
+$(function(){
+    init_page_js();
+});
+
+function refresh_plot(){
+    var params = get_parameters();
+    if (window.current_plot_data == null){
+        get_plot_data(key=params[0], name=params[1], view_type=params[2], search_string=params[3], inst_type_filter=params[4]);
+    }else{
+        make_plot(key=params[0], name=params[1], view_type=params[2], inst_type_filter=params[4]);
+    }
+}
+
+function make_plot(key, name, view_type, filter_inst_type){
+    var toplot = {
+        chart: {
+            type: 'scatter',
+            events: {
+                render: function(){
+                    let series = this.series
+                    let sum = 0
+                    for (let i = 0; i < series.length; i++){
+                        if (series[i].visible){
+                            for (let j = 0; j < series[i].data.length; j++){
+                                sum += series[i].data[j].y
+                            }
+                        }
+                    }
+                    if (sum === 0) return '0 Bytes';
+                    const k = 1024;
+                    const dm = 2 < 0 ? 0 : 2;
+                    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+                    const i = Math.floor(Math.log(sum) / Math.log(k));
+                    var c_sum = parseFloat((sum / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+                this.setTitle({text: "File size of projects, sum in bytes: " + c_sum}, false, false)
+                }
+            }
+        },
+        title: {
+            text : name+' of projects'
+        },
+        yAxis: {
+            min : 0,
+            title : {
+              text : name
+            }
+        },
+        legend: {
+            title: {
+                text: 'Click to hide:',
+                align: 'center'
+                }
```

Comment:
Pedantic, I know - it does make it easier to read though :) You also have a mix of 2- and 4-space indentation, but that bothers me less.

```suggestion
            }
```